Pod/Classes/PINRemoteImageManager.m

```diff
@@ -1241,6 +1244,11 @@ - (void)pin_addOperationWithQueuePriority:(PINRemoteImageManagerPriority)priorit
 {
     NSBlockOperation *operation = [NSBlockOperation blockOperationWithBlock:block];
     operation.queuePriority = operationPriorityWithImageManagerPriority(priority);
+#if defined(__IPHONE_8_0)
+    operation.qualityOfService = NSOperationQualityOfServiceBackground;
+#else
+    operation.threadPriority = 0.2;
```

Comment:
So we only set the thread priority for iOS 7, correct?  If we assigned a thread priority unconditionally, would that conflict with our QoS preference on iOS 8+?